src/audits/ImageWithoutAltText.js

```diff
@@ -27,7 +28,16 @@ axs.AuditRules.addRule({
             !axs.utils.isElementOrAncestorHidden(element);
     },
     test: function(image) {
-        return (!image.hasAttribute('alt') && image.getAttribute('role') != 'presentation');
+        var imageIsPresentational = (image.hasAttribute('alt') && image.alt == "") || image.getAttribute('role') == 'presentation';
```

Comment:
total nit, question of consistency - since the strings are all single quoted should the empty string test be against `''` instead of `""`?